templates/insights/blocks/promoted_item.html

```diff
@@ -34,9 +34,11 @@ <h3 class="blog-embed__heading">
             {% image value.teaser_image min-690x260 alt=value.teaser_alt_text class="blog-embed__image" %}
         </div>
 
-        <p class="blog-embed__text">
-            {{ value.description|richtext }}
-        </p>
+        <div class="blog-embed__text">
+            <p>
```

Comment:
Because the description field is rendered as rich text, it's likely to contain `p` elements which the browser is going to interpret as:

```html
<div class="blog-embed__text">
    <p></p>
    <p>Rich</p>
    <p>text</p>
    <p></p>
</div>
```

Reply:
Thanks Dan, I've removed that `p` tag now 👍